dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetVersionDifference.java

```diff
@@ -964,22 +997,130 @@ public void setFile2Empty(boolean state) {
             file2Empty = state;
         }
 
+    }
+    
+    public class FileSummary {
```

Comment:
I wonder if we should not already extract those classes to 'outside' when we are at it. At the same time we could also move the DatasetVersionDifference to proper package. One less task in the future refactoring. If those classes are only used here they can also be package private.

Reply:
ok